lib/modules/datasource/maven/util.ts

```diff
@@ -454,7 +458,10 @@ export async function getDependencyInfo(
       }
 
       const sourceUrl = pomContent.valueWithPath('scm.url');
-      if (sourceUrl && !containsPlaceholder(sourceUrl)) {
+      if (
+        sourceUrl &&
+        !containsPlaceholder(removeKnownPlaceholders(sourceUrl))
+      ) {
```

Comment:
```suggestion
const sourceUrl = removeKnownPlaceholders(pomContent.valueWithPath('scm.url'));
      if (sourceUrl && !containsPlaceholder(sourceUrl)) {
```

Oh no, it's not stripped in final result 🙈

Reply:
oh, it's cleaned later

https://github.com/renovatebot/renovate/blob/e477026da21959f8f1455e18557458734eb88ede/lib/modules/datasource/metadata.ts#L53